pkg/watermark/otbucket/otbucket.go

```diff
@@ -0,0 +1,32 @@
+package otbucket
```

Comment:
open to better names..

Reply:
just ot? it is anyways encapsulating OTValue